README.md

```diff
@@ -109,4 +109,116 @@ Works that are forked from other projects are under their original licenses.
 
 ## Credit
 
-The general structure of `refmt` repo was copied from [whitequark's m17n project](https://github.com/whitequark/ocaml-m17n), including parts of the `README` that instruct how to use this with the OPAM toolchain. Thank you OCaml!
+The general structure of `refmt` repo was copied from [whitequark's m17n project](https://github.com/whitequark/ocaml-m17n), including parts of the `README` that instruct how to use this with the OPAM toolchain. Thank you, OCaml!
```

Comment:
Copied from here https://github.com/facebook/reason/tree/master#installation-for-programmatic-usage ?